exp/templates/exp/_navigation.html

```diff
@@ -7,7 +7,12 @@
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
       </button>
-      <a class="navbar-brand" href="{% url 'exp:study-list' %}">Experimenter</a>
+      {% if request.user.groups.exists %}
```

Comment:
You could also check `request.user.is_researcher` and then send them to `exp:dashboard` that way if they wanted to check their status and they hadn't been approved yet they could still get the purgatory page telling them to wait.